arch/tests/bootstrap/test_bootstrap.py

```diff
@@ -545,6 +545,59 @@ def test_jackknife(self):
         results_series = _loo_jackknife(self.func, len(y), (y,), {})
         assert_allclose(results, results_series)
 
+    def test_bca_against_bcajack(self):
+        import rpy2.rinterface as ri
+        import rpy2.robjects as robjects
+        import rpy2.robjects.numpy2ri
+        from rpy2.robjects.packages import importr
+        rpy2.robjects.numpy2ri.activate()
+        utils = importr('utils')
+        try:
+            bcaboot = importr('bcaboot')
+        except Exception:
+            utils.install_packages('bcaboot', repos='http://cran.us.r-project.org')
+            bcaboot = importr('bcaboot')
+        observations = np.random.multivariate_normal(mean=[8, 4], cov=np.identity(2), size=20)
```

Comment:
You should use 

```
rs = np.random.RandomState(seed)
rs.multivariate_normal(mean=[8, 4], cov=np.identity(2), size=20)
```

to generate the data for some seed value.  

These values can then be exported to R to compute any values for comparison.